src/main/java/ee/buerokratt/ruuter/domain/steps/http/HttpDefaultService.java

```diff
@@ -0,0 +1,25 @@
+package ee.buerokratt.ruuter.domain.steps.http;
+
+import ee.buerokratt.ruuter.domain.ConfigurationInstance;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.slf4j.MDC;
+
+import java.util.HashMap;
+
+@Data
+@NoArgsConstructor
+public class HttpDefaultService {
+    private String service;
+    private HashMap<String, Object> body;
+    private HashMap<String, Object> query;
+
+    public void executeHttpDefaultAction(ConfigurationInstance ci, String resultName) {
+        HttpQueryResponse response = ((HttpStepResult) ci.getContext().get(resultName)).getResponse();
+        body = body == null ? new HashMap<>() : body;
```

Comment:
This can be remove if you instantiate the body with an empty map on line 14:
`private HashMap<String, Object> body = new HashMap<>();`